plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1421,12 +1421,25 @@ public void testTimestampCoercion()
                 .addRoundTrip("TIMESTAMP '1969-12-31 23:59:59.999999499999'", "TIMESTAMP '1969-12-31 23:59:59.999999'")
                 .addRoundTrip("TIMESTAMP '1969-12-31 23:59:59.9999994'", "TIMESTAMP '1969-12-31 23:59:59.999999'")
 
-                // CTAS with Trino, where the coercion is done by the connector
                 .execute(getQueryRunner(), trinoCreateAsSelect("test_timestamp_coercion"))
-                // INSERT with Trino, where the coercion is done by the engine
                 .execute(getQueryRunner(), trinoCreateAndInsert("test_timestamp_coercion"));
     }
 
+    @Test
+    public void testTimestampCoercionShowCreateTable()
```

Comment:
Can we move this test to `PostgresConnectorTest` ? Can we also have additional test coverage where we run `CREATE TABLE ` with a timestamp column of higher precision ?